instrumentation/lettuce-5.0/src/main/java/io/lettuce/core/RedisClient.java

```diff
@@ -0,0 +1,32 @@
+package io.lettuce.core;
+
+import com.newrelic.api.agent.weaver.Weave;
+import com.newrelic.api.agent.weaver.Weaver;
+import io.lettuce.core.api.StatefulRedisConnection;
+import io.lettuce.core.codec.RedisCodec;
+import io.lettuce.core.resource.ClientResources;
+
+import java.time.Duration;
+
+@Weave(originalName = "io.lettuce.core.RedisClient")
+public abstract class RedisClient extends AbstractRedisClient {
```

Comment:
Same comment about naming weaved library classed with `_Instrumentation` and specifying the `originalName` in the `Weave` annotation.